package.json

```diff
@@ -32,7 +32,7 @@
 		"lint:fix": "eslint --fix \".*.js\" \"**/*.{js,ts,tsx}\"",
 		"lint:quiet": "eslint --quiet \".*.js\" \"**/*.{js,ts,tsx}\"",
 		"lint": "eslint \".*.js\" \"**/*.{js,ts,tsx}\"",
-		"site": "cd next.clayui.com && yarn build && yarn serve",
+		"site": "cd clayui.com && yarn build && yarn serve",
```

Comment:
Thanks for fixing this although it has nothing to do with the issue.

Reply:
I used it to test the issue following Georgel Pop instructions and it seems the path was wrong.

Reply:
Yes, exactly. Good catch